pkg/transport/manager.go

```diff
@@ -478,17 +478,6 @@ func (tm *Manager) DeleteTransport(id uuid.UUID) {
 
 	// Deregister transport before closing the underlying transport.
 	if tp, ok := tm.tps[id]; ok {
-		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-		defer cancel()
-
-		// todo: this should probably be moved to tp.close because we want to deregister
-		// a transport completely and not deal with transport statuses at all
```

Comment:
Are we deregistering anywhere else upon shutdown that I am missing? Seems like we are not removing tps from discovery at all if we delete this.

Reply:
We are already deregistering inside tp.close as the todo is sugesting to do. [tp.close()](https://github.com/ersonp/skywire/blob/ef9b1dcd93c7573a30068ac17cba906a2e5de1b2/pkg/transport/managed_transport.go#L192)
[deleteFromDiscovery()](https://github.com/ersonp/skywire/blob/ef9b1dcd93c7573a30068ac17cba906a2e5de1b2/pkg/transport/managed_transport.go#L334) is called inside tp.close()